demo/readme.md

```diff
@@ -1,8 +1,10 @@
 # Demo set up for Suitest JS API with Mocha test runner
 
-1. Copy `testConfig.js.dist` to `testConfig.js`.
-2. Fill in fields with necessary data.
-3. Set automatedSession to `true` if you want to start test pack and run test in automated session mode.
-4. Update or remove `test/dummy.test.js`, optionally create more tests in that folder.
-5. In terminal navigate to `demo` folder and run `yarn` or `npm i`.
-6. Run `yarn test` or `npm test`.
+1. You may want create config file instead of passing params directly in console
```

Comment:
1. Copy `.suitestrc.dist` file as `.suitestrc` and fill in it's contents with your values according to [documentation](https://suite.st/docs/suitest-api/setup/#environment-setup).
2. Write your tests in `test/` folder. There is an example in `test/dummy.test.js`.